index.js

```diff
@@ -3,6 +3,6 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+/* tslint:disable:no-unused-variable */
 const nodeMajorVersion = +process.versions.node.split('.')[0];
```

Comment:
no need for this line

Reply:
I thought we were keeping these lines in so that once Node 10 hits, we can reintroduce version-splitting?

Reply:
Yea that was the original idea @kjdelisle, but we decided to get rid of it, and re-introduce it when Node 10 hits.

Reply:
I'll remove it to be consistent with lb-next, we should be able to re-add it easily :).